hw/ip/otbn/data/bignum-insns.yml

```diff
@@ -774,13 +774,19 @@
   syntax: |
     <grd>[<grd_inc>], <offset>(<grs1>[<grs1_inc>])
   doc: |
-    Calculates a byte memory address by adding the offset to the value in the GPR `grs1`.
-    The value from this memory address is then copied into the WDR pointed to by the value in GPR `grd`.
+    Load a WLEN-byte little-endian value from memory.
+
+    The load address is `offset` plus the value in the GPR `grs1`.
+    The loaded value is stored into the WDR given by the bottom 5 bits of the GPR `grd`.
 
     After the operation, either the value in the GPR `grs1`, or the value in `grd` can be optionally incremented.
 
-    - If `grs1_inc` is set, the value in `grs1` is incremented by the value WLEN/8 (one word).
-    - If `grd_inc` is set, the value in `grd` is incremented by the value 1.
+    - If `grs1_inc` is set, the value in `grs1` is incremented by value WLEN/8 (one word)
+    - If `grd_inc` is set, the value in `grd` is incremented by one.
+
+    Register increments are performed with wrapping.
+    If `grs1` is incremented, the value will be wrapped modulo the size of DMEM.
```

Comment:
I'm not happy with this behavior, continuing the discussion in https://github.com/lowRISC/opentitan/issues/3587

Reply:
I understand. Can we get this change in independent of that discussion for now? The behaviour without it is very wrong, and this patch gets us to "arguably wrong" :-p